src/Dev/DebugView.php

```diff
@@ -30,38 +30,7 @@ class DebugView
     private static $columns = 100;
 
     protected static $error_types = [
-        0 => [
```

Comment:
These values aren't being used anywhere, only the errno-mapped ones are. And 2 conflicted with E_USER_WARNING